content/contributing/reference/triage.md

```diff
@@ -0,0 +1,126 @@
++++
+title = "Triage"
+insert_anchor_links = "right"
+[extra]
+weight = 1
++++
+
+Bevy's scope and user base means that it gets a large volume of issues and pull requests. This work all needs to be organized: making it easier to search through to find related issues, to figure out where you can help, and to quickly communicate the status of a task at a glance. The continual process of organizing work is called **triage**.
```

Comment:
this is a minor nitpick to simplify this sentence a bit and I'm cool with it either way (I realize this was originally suggested by Alice)

>This work all needs to be organized to make it easier to find related issues, figure out where you can help, and to quickly communicate the status of a task at a glance.

Reply:
Yep, I'm on board with that.